index.html

```diff
@@ -212,18 +219,21 @@ <h3>Skills</h3>
         </ul>
     </section>
     <footer>
```

Comment:
- [ ] please kindly reduce the space between your skill section cards 
your design 
![Screenshot (97)_LI](https://user-images.githubusercontent.com/65298048/171514301-e11b3d44-5f56-43e1-98fc-52915010e3bb.jpg)
figma design
![Screenshot (99)](https://user-images.githubusercontent.com/65298048/171514323-e4fed5e6-413d-444d-a4a5-c72fafe57cad.png)